bot/potentialScammer/getTargetUserNameList.js

```diff
@@ -0,0 +1,13 @@
+const getTargetUsernameList = async (client, guild) => {
```

Comment:
Return both username and user id.
Because user id is unique, I guess we will need it when want to ban scammer.